addons/calendar/models/res_users.py

```diff
@@ -10,6 +10,12 @@
 class Users(models.Model):
     _inherit = 'res.users'
 
+    calendar_user_settings = fields.Many2one('calendar.user.settings')
+
+    _sql_constraints = [
+        ('user_settings_uniq', 'unique (calendar_user_settings)', "The user has already calendar settings"),
+    ]
```

Comment:
The error message seems wrong, It prevent a user setting to be set to 2 calendar so it should be "These settings already belong to an user" or smth like that